docs/book/v2/standard-filters.md

```diff
@@ -1912,6 +1912,9 @@ as the result.
 
 ## UriNormalize
 
+CAUTION: **Deprecated**
+This filter is deprecated since version 2.36.0 and will be remove in version 3.0. There will not be a replacement.
```

Comment:
> There will not be a replacement.

I think there should be a new filter for this?

Reply:
Does #133 go far enough here? If so, I'll amend to reference this one after merging